ODSReader.py

```diff
@@ -66,10 +66,11 @@ def readSheet(self, sheet):
                 # for each text/text:span node
                 for p in ps:
                     for n in p.childNodes:
-                        if (n.nodeType == 1 and n.tagName == "text:span"):
+                        if (n.nodeType == 1 and
+                                ((n.tagName == "text:span") or (n.tagName == "text:a"))):
                             for c in n.childNodes:
                                 if (c.nodeType == 3):
-                                    textContent = u'{}{}'.format(textContent, n.data)
+                                    textContent = u'{}{}'.format(textContent, c.data)
```

Comment:
Thanks for opening this pull request @wayne-butterworth!

Have you tested that this change from `n.data` to `c.data` still works with normal text nodes?

Reply:
Yes, it does. Do you want me to attach my test data?

Reply:
This fix should be merged. 
It fixes issue #9